modules/openapi-generator/src/main/resources/python/model_anyof.mustache

```diff
@@ -24,16 +28,16 @@ class {{classname}}({{#parent}}{{{.}}}{{/parent}}{{^parent}}BaseModel{{/parent}}
     {{vendorExtensions.x-py-name}}: {{{vendorExtensions.x-py-typing}}}
 {{/composedSchemas.anyOf}}
     if TYPE_CHECKING:
-        actual_instance: Union[{{#anyOf}}{{{.}}}{{^-last}}, {{/-last}}{{/anyOf}}]
+        actual_instance: Optional[Union[{{#anyOf}}{{{.}}}{{^-last}}, {{/-last}}{{/anyOf}}]] = None
```

Comment:
@multani Hi Jonathan! Do you remember, by any chance, why do you introduce Optional here? Is it possible to have None value here even with one_of validator?

Reply:
Hey :wave: Sorry, I don't remember the details of this particular change :/ Does it create a problem for you?

Reply:
Yes, unfortunately. Before this changes non-optional anyOf had correct typing (Union[A, B]), but now it has additional None in the union, which isn't right and stops me from using proper typing. Should I try to change it back and create PR for it?

Reply:
Yes, of course, feel free to create a pull request!

I just checked again, and there's also a `is None` check in the `to_dict` method at the end of the file, it's probably why I added the `Optional` type in the first place. If you remove `Optional`, can you also remove that check?